pom.xml

```diff
@@ -247,6 +247,8 @@
               <goal>shade</goal>
             </goals>
             <configuration>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
```

Comment:
This will also affect the IDEA plugin **and** the parent (it generates a `google-java-format-parent-0.1-SNAPSHOT-deploy.pom` file as an empty zip archive that *will* be deployed to Central).

IMO, that plugin (unless needed for the IDEA plugin too) should be moved to `core/pom.xml` (or possibly to `<pluginManagement>` and then added explicitly –without configuration– into `core/pom.xml`)

Otherwise, yes, the generated JAR for `core` is OK.